exporter/collector/metrics.go

```diff
@@ -115,12 +116,15 @@ type metricMapper struct {
 	normalizer normalization.Normalizer
 	obs        selfObservability
 	cfg        Config
+	// self-observability metrics
+	exemplarAttachmentDropCount metricapi.Int64Counter
```

Comment:
would this fit better under `obs` than its own field in the metricMapper?

Reply:
obs is shared by the metrics, logs, and trace exporters, so it didn't seem like the right place to put it.